lib/lita/adapters/slack.rb

```diff
@@ -9,7 +9,9 @@ class Slack < Adapter
       # Required configuration attributes.
       config :token, type: String, required: true
       config :proxy, type: String
-      config :parse, type: String
+      config :parse, type: [String, NilClass]
+      config :link_names, type: [FalseClass, TrueClass, NilClass]
```

Comment:
fak u slack

in that case, just leave off the default option. You don't have to specify nil as a valid type because the types only get checked when the value is explicitly assigned, and all config values default to nil.